userpanel/modules/documents/style/bclean/templates/documents.html

```diff
@@ -130,23 +126,17 @@ <h4 class="modal-title">{trans("Document submission")}</h4>
 							{foreach $doc.attachments as $docattach}
 								<div class="row document-attachment">
 									<div class="col-md-auto">
-										<a class="btn btn-primary" href="?m=documents&f=docview&id={$doc.id}&attachmentid={$docattach.id}" target="_blank">
 											{if $docattach.type == 1}
-												{trans("View")}
+												{button class="btn btn-primary" href="?m=documents&f=docview&id={$doc.id}&attachmentid={$docattach.id}" external="_blank" icon="view" label="View"}
 											{else}
-												{$docattach.filename}
+												{button class="btn btn-primary" href="?m=documents&f=docview&id={$doc.id}&attachmentid={$docattach.id}" external="_blank" icon="document" label="{$docattach.filename}"}
```

Comment:
` external="_blank" ` => ` external=true `.

Reply:
Zrobione to i inne wystąpienia.